aten/src/ATen/native/cuda/BinaryOpsKernel.cu

```diff
@@ -44,7 +38,7 @@ void div_kernel_cuda(TensorIterator& iter) {
       });
     });
   } else {
-    AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "div_cuda", [&]() {
+    AT_DISPATCH_ALL_TYPES_AND(kHalf, iter.dtype(), "div_cuda", [&]() {
```

Comment:
Are we moving towards using `AT_DISPATCH_ALL_TYPES_AND(kHalf, ...)` instead of `AT_DISPATCH_FLOATING_TYPES_AND_HALF`?

Reply:
I think so:

https://github.com/pytorch/pytorch/blob/adbab82846d1b6fb6fb4cb395df57a4c45c91335/aten/src/ATen/Dispatch.h#L60-L61

Reply:
Thanks for pointing that out